SI/Sem.03/Pract.03/Task04.cpp

```diff
@@ -0,0 +1,26 @@
+#include <iostream>
+
+using namespace std;
+
+int main()
+{
+	int m, n;
+	cin >> m >> n;
+	if (m <= 0 || n <= 0)
+	{
+		cout << "m and n must be larger than 0";
+	}
+	else
+	{
+		int p = m * n;
+
+		for (int i = 0; i < p; i++)
```

Comment:
we can start from 1 and condition will change to <=.
In addition, p is not necessary.